Remove requirements for wheel and pip as they're not required for build

--- setup.cfg.orig	2022-01-19 06:16:57 UTC
+++ setup.cfg
@@ -32,7 +32,7 @@ classifiers = 
 zip_safe = False
 include_package_data = True
 packages = find:
-setup_requires = setuptools;wheel;pip
+setup_requires = setuptools
 python_requires = >=3.7
 tests_require = django
 
